docs/www/quick-start-kubernetes.md

```diff
@@ -4,91 +4,178 @@ order: 0
-kind create cluster
-```
+  </tab>
 
-Once the cluster is created please follow the [Helm install instructions](#Using-Helm-to-Install-Redpanda).
+  <tab id="AWS EKS">
 
-## AWS EKS
+  Use the [EKS Getting Started](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) guide to set up EKS.
+  When you finish, you'll have `eksctl` installed so that you can create and delete clusters in EKS.
+  Then, create an EKS cluster with:
 
-First complete all the steps describe in [EKS Getting Started](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html)
-guide. This includes installing `eksctl` which is used to create and delete
-clusters in EKS.
+  ```
+  eksctl create cluster \
+  --name redpanda \
+  --nodegroup-name standard-workers \
+  --node-type m5.xlarge \
+  --nodes 3 \
+  --nodes-min 1 \
+  --nodes-max 4 \
+  --node-ami auto
+  ```
 
-Once `eksctl` is installed, you can then use it to create an EKS cluster:
+  It will take about 10-15 minutes for the process to finish.
 
-```
-eksctl create cluster \
---name redpanda \
---nodegroup-name standard-workers \
---node-type m5.xlarge \
---nodes 3 \
---nodes-min 1 \
---nodes-max 4 \
---node-ami auto
-```
+  </tab>
 
-This command will take 10-15 minutes to complete. Once it is completed then
-follow the [Helm install instructions](#Using-Helm-to-Install-Redpanda).
+  <tab id="Google GKE">
 
-## Google GKE
+  First complete the "Before You Begin" steps describe in [Google Kubernetes Engine Quickstart](https://cloud.google.com/kubernetes-engine/docs/quickstart).
+  Then, create a cluster with:
 
-First complete the "Before You Begin" steps describe in
-[Google Kubernetes Engine Quickstart](https://cloud.google.com/kubernetes-engine/docs/quickstart).
-Once complete you can create a cluster using the following command:
+  ```
+  gcloud container clusters create redpanda --machine-type n1-standard-4
+  ```
 
-```
-gcloud container clusters create redpanda --machine-type n1-standard-4
-```
+  </tab>
+</tabs>
+
+## Using Helm to Install Redpanda
 
-Once the command completes then follow the
-[Helm install instructions](#Using-Helm-to-Install-Redpanda)
\ No newline at end of file
+1. Using Helm, add the Redpanda chart repository and update it:
+
+    ```
+    helm repo add redpanda https://charts.vectorized.io/ && helm repo update
+    ```
+
+2. Install the Redpanda operator CRDs:
+
+    ```
+    kubectl apply -k 'https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd?ref=<latest version>'
+    ```
+
+    You can find the latest version number of the operator on the [list of operator releases](https://github.com/vectorizedio/redpanda/releases).
+
+3. Install Redpanda on your Kubernetes cluster with:
+
+    ```
+    helm install --namespace redpanda-system --create-namespace redpanda-operator redpanda/redpanda-operator
+    ```
+
+## Next steps
+
+After you set up Redpanda in your Kubernetes cluster, you can:
+
+- Install resources from our sample files:
```

Comment:
I would add one more step that creates some playground namespace e.g.`test-namespace` or `redpanda-test` 

```
kubectl create ns redpanda-test
```